projects/hard.js

```diff
@@ -18,7 +19,8 @@ const coingeckoIds = {
     [BUSD_DENOM]:'binance-usd',
     [XRPB_DENOM]:'ripple',
     [ATOM_DENOM]:'cosmos',
-    [AKT_DENOM]:'akash-network'
+    [AKT_DENOM]:'akash-network',
+    [AXLUSDC_DENOM]: 'axlusdc'
```

Comment:
Is this the correct source for this id for the price? https://www.coingecko.com/en/coins/axelar-usdc

Reply:
yes, that looks right.